### ISSUE TITLE
Doesn't send Series name to TorzNab

### ISSUE THREAD

**Issue by Inrego** [Sonarr/Sonarr#849]
Since a recent update, Sonarr no longer sends the series name to Torznab. That means when I do manual search, I get lots of wrong results (any series with the same season-episode).

This is the URI parameters being sent to TorzNab:
t=tvsearch&cat=5030,5040&extended=1&apikey=*****&offset=0&limit=100&season=6&ep=3
![](http://i.imgur.com/r77R1QY.png)

**Comment by Taloth** [comment]
Will be fixed momentarily, see https://forums.sonarr.tv/t/develop-torznab-searching/7451

**Comment by Inrego** [comment]
I'm still experiencing this in some cases. For example "How to get away with murder". No q is being sent to torznab

**Comment by Inrego** [comment]
I'm still experiencing this in some cases. I have 2 torznab indexers. I'm searching for an episode in How to get away with murder

Here are the urls for each request:

/api?t=tvsearch&cat=5030,5040&extended=1&apikey=lgr4hvvj8jml56ep9wo7a58aggh4xfs1&offset=0&limit=100&q=How to Get Away with Murder&season=2&ep=5

/api?t=tvsearch&cat=5030,5040&extended=1&apikey=lgr4hvvj8jml56ep9wo7a58aggh4xfs1&offset=0&limit=100&rid=37307&season=2&ep=5

For one indexer, it sends q, for the other it sends rid.

**Comment by markus101** [comment]
Its not sending the title because the RID is being sent instead, only one is sent.

Its sending the RID instead of the title because Torznab told Sonarr that it supports searching by RID.

**Comment by Inrego** [comment]
Both indexers were provided by a middleware called Jackett, so they should both provide the same properties.

I removed all my indexers, and readded them, and then it started sending it right. I have a theory, that I had accidentally added one of them as a newznab indexer at first, then removed it and readded it as torznab. I think it had some of that info cached that it wouldn't let go of even if I removed and readded that particular indexer. Not until I removed all and added them again.
